### DISCUSSION TITLE
open in explorer command from search view

### DISCUSSION THREAD

**Discussion by danielbsweeney** [tadashi-aikawa/obsidian-another-quick-switcher#153] [Ideas]
hi,

great plugin.

just wondering if there would be a chance we could have a open in explorer command when searched. open in default app is there but i don't believe there is a way to open in explorer.

cheers

**Comment by tadashi-aikawa** [comment]
@danielbsweeney 
Sorry for the late answer 🙇 

It would be possible. I'll convert this discussion to an issue and consider it.
